src/datasources/human-description-api/human-description-api.service.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { HumanDescriptionApi } from './human-description-api.service';
+
+describe('HumanDescriptionAPI', () => {
+  const contractDescriptions = {
+    'function transfer(address, uint256)':
+      'Send {{tokenValue 1}} to {{address 0}}',
+  };
+
+  it('should return parsed messages from json', () => {
+    const humanDescriptionApi = new HumanDescriptionApi(contractDescriptions);
+
+    const parsedMessages = humanDescriptionApi.getParsedMessages();
+
+    expect(parsedMessages['function transfer(address, uint256)']).toEqual({
+      process: expect.anything(),
```

Comment:
Let's make this more robust.

```suggestion
      process: expect.any(Function),
```